src/common.js

```diff
@@ -236,6 +244,7 @@ exports.randomFileName = randomFileName;
 // extend(target_obj, source_obj1 [, source_obj2 ...])
 // Shallow extend, e.g.:
 //    extend({A:1}, {b:2}, {c:3}) returns {A:1, b:2, c:3}
+// XXX: This isn't going to be a part of plugin.utils. We should upgrade to Babel, Then we'll use Object.assign (or a ponyfill)
```

Comment:
I think we should upgrade to babel and use Object.assign, or we could just use [a polyfill](https://www.npmjs.com/package/object-assign)